gusto/configuration.py

```diff
@@ -134,6 +135,33 @@ class ShallowWaterParameters(Configuration):
     H = None  # mean depth
 
 
+class EadyParameters(Configuration):
+
+    """
+    Physical parameters for Incompressible Eady
+    """
+    Nsq = 2.5e-05  # squared Brunt-Vaisala frequency (1/s)
```

Comment:
It's probably a bit dangerous to have N and Nsq both in the parameters with inconsistent values.